azure-pipelines/Get-NuGetTool.ps1

```diff
@@ -6,7 +6,7 @@
 #>
 Param(
     [Parameter()]
-    [string]$NuGetVersion='5.2.0'
+    [string]$NuGetVersion='latest'
```

Comment:
Thanks, but this repo doesn't do "latest" because that tends to introduce breaks to repos based on this template at unpredictable times and investigation costs are quite high before someone realizes that a version pushed to the cloud by another org broke something. I appreciate the update, but can you change this to the latest version number rather than `latest`?

Reply:
Updated the version to 6.4.0 which is latest as of today.